src/mantarray_waveform_analysis/pipelines.py

```diff
@@ -200,13 +202,13 @@ def get_magnetic_data_metrics(
             ],
         ],
     ]:
-        """Calculate data metrics on noise filtered magnetic data."""
+        """Calculate data metrics on force data."""
         try:
             return self._magnetic_data_metrics
         except AttributeError:
             pass
         self._magnetic_data_metrics = data_metrics(
-            self.get_peak_detection_results(), self.get_noise_filtered_magnetic_data()
+            self.get_peak_detection_results(), self.get_compressed_force()
```

Comment:
Is there a reason you're using the "compressed force" to calculate the metrics instead of regular force?

Reply:
The reason we have compressed data is to be able to pass the raw x/y data stream to the front-end to be able to be animated at a fast frame rate.  But for accuracy of the metrics we typically want to do all our calculations on the non-compressed datastream